quranHome/src/main/res/layout/quran_home_view.xml

```diff
@@ -34,13 +34,28 @@
                     android:layout_height="match_parent">
 
                     <TextView
-                        android:id="@+id/txtSearchName"
+                        android:id="@+id/txtJuzTitle"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:layout_alignParentStart="true"
                         android:layout_centerVertical="true"
                         android:fontFamily="@font/montserrat_bold"
-                        android:text="Reyan"
+                        android:paddingStart="8dp"
+                        android:paddingEnd="8dp"
+                        android:text="جزء"
+                        android:textColor="@color/yellow"
+                        android:textSize="@dimen/extra_large_text"
+                        android:textStyle="bold"
+                        tools:ignore="HardcodedText,SpUsage" />
+
+                    <TextView
+                        android:id="@+id/txtIndexTitle"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_centerHorizontal="true"
+                        android:layout_centerVertical="true"
+                        android:fontFamily="@font/montserrat_bold"
+                        android:text="فهرست سوره‌ها"
```

Comment:
please use `string` resources

Reply:
Sure, because of being draft and temporary.